packages/design-system-react/tsconfig.build.json

```diff
@@ -5,15 +5,9 @@
     "outDir": "./dist",
     "rootDir": "./src",
     "jsx": "react",
-    "types": [
-      "react",
-      "jest",
-      "@testing-library/react",
-      "@testing-library/jest-dom"
-    ],
+    "types": ["react"],
     "lib": ["ES2020", "DOM"]
   },
   "references": [],
-  "include": ["./src"],
-  "exclude": ["**/*.stories.tsx", "**/*.stories.ts"]
```

Comment:
Storybook files are excluded from the root `tsconfig.packages.build.json` so it's not needed here